bfe-verso-profiles/profiles/BIBFRAME 2.0 Prints and Photographs.json

```diff
@@ -720,8 +732,7 @@
               "useValuesFrom": [],
               "valueDataType": {},
               "editable": "false",
-              "repeatable": "false",
-              "defaultLiteral": "Title devised by Library staff"
```

Comment:
since this defaultLiteral has no defaultURI to accompany it, and since it seems like an exemplar more than a default, I removed it.  It could be kept if a defaultURI was added.  (I looked for a URI for it, but didn't find one).